src/test/cc/math/distributed_discrete_gaussian_noiser_test.cc

```diff
@@ -0,0 +1,47 @@
+// Copyright 2020 The Cross-Media Measurement Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "math/distributed_discrete_gaussian_noiser.h"
+
+#include <unordered_map>
+
+#include "common_cpp/macros/macros.h"
+#include "common_cpp/testing/status_macros.h"
+#include "common_cpp/testing/status_matchers.h"
+#include "gtest/gtest.h"
+
+namespace wfa::math {
+namespace {
+DistributedDiscreteGaussianNoiser distributedGaussianRandomNoise(
+    {.sigma = 0.73});
+
+TEST(DiscreteGaussianNoiseIndividualComponent, StatusIsOK) {
+  ASSERT_THAT(distributedGaussianRandomNoise.GenerateNoiseComponent(), IsOk());
+}
+
+TEST(DiscreteGaussianNoiseGenerateNoiseComponent, ReturnsSampledValues) {
+  int num = 5;
+  std::vector<int64_t> results;
+
+  for (size_t i = 0; i < num; ++i) {
+    ASSERT_OK_AND_ASSIGN(
+        int64_t temp, distributedGaussianRandomNoise.GenerateNoiseComponent());
+    results.push_back(temp);
+  }
+
+  ASSERT_THAT(results.size(), num);
+}
+
```

Comment:
Maybe we should also add a statistical test similar to the one for distributed geometric noise to make sure we generate the noise correctly?

Reply:
Pinging on this comment. Also, since we now have `contributor_count`, we should perhaps add tests for the case `contributor_count` > 1 too?